common/src/main/public/development.conf.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I'm guessing we need to have a `conf.json` now since there's no instance of copying `development.conf.json`. Would a symbolic link work, so we don't have duplicate things?

Reply:
This is in the TODO. I need to somehow proxy this but I'm not there yet, this is a workaround.